samples/subsys/usb/mass/prj_nrf52840_pca10056.conf

```diff
@@ -3,10 +3,24 @@ CONFIG_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
 CONFIG_USB_DEVICE_PRODUCT="Zephyr MSC sample"
+CONFIG_USB_MASS_STORAGE=y
 CONFIG_LOG=y
+
 CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
-CONFIG_USB_MASS_STORAGE=y
 CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
 CONFIG_USB_MASS_STORAGE_LOG_LEVEL_ERR=y
+CONFIG_DISK_LOG_LEVEL_ERR=y
+
 CONFIG_DISK_ACCESS_RAM=y
-CONFIG_MASS_STORAGE_DISK_NAME="RAM"
+
+CONFIG_MASS_STORAGE_DISK_NAME="NAND"
+CONFIG_DISK_ACCESS_FLASH=y
+CONFIG_DISK_FLASH_DEV_NAME="NRF_FLASH_DRV_NAME"
+CONFIG_DISK_FLASH_START=0xf8000
```

Comment:
Where does it point? Please use DT_FLASH_AREA_STORAGE_OFFSET and maybe DT_FLASH_AREA_STORAGE_SIZE.

Reply:
@masz-nordic in order to do that use the python function as seen here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/Kconfig.zephyr#L93

Reply:
@carlescufi Are those functions callable from `.conf` files? They are used exclusively in `Kconfig` files and when trying to use one I get an error:
```
Error: Aborting due to non-whitelisted Kconfig warning 'warning: the value
'$(dt_hex_val,DT_FLASH_AREA_STORAGE_OFFSET)' is invalid for DISK_FLASH_START (defined at
subsys/disk/Kconfig:75), which has type hex -- assignment ignored'. Note: If this warning doesn't
point to an actual problem, you can add it to the whitelist at the top of
/home/marcin/ncs/zephyr/scripts/kconfig/kconfig.py.
```

Reply:
ah, no sorry. Those are only valid for `Kconfig` files, so I don't see any way of doing it other than hardcoding the value unless you can get `DISK_FLASH_START` to be correctly configured without using the `.conf` file?

Reply:
>Where does it point? Please use DT_FLASH_AREA_STORAGE_OFFSET and maybe DT_FLASH_AREA_STORAGE_SIZE.

You can't do that in `.conf` files at all, you can only use constants to assign.

Reply:
Sorry, did not think about it, any way to make more generic?

Reply:
@jfischer-phytec-iot The only way to make it more generic is to not set it in the `.conf` but rather have it set automatically by a `defaul` in a `Kconfig` file. Unless that is possible, I would leave as-is.